### ISSUE TITLE
Keyboard Functionality

### ISSUE THREAD

**Issue by cptvitamin** [sparksuite/simplemde-markdown-editor#195]
Use Case: Keyboard users should be able to use the editor.
Currently, there is no way to exit the editor without using the mouse. The TAB key inserts a tab. AFAIK, there is no legitimate reason to insert tabs in a markdown document.  I propose that TAB and SHIFT+TAB move focus in to and out of the editor.

Ideally, the toolbar will also have keyboard accessibility.

**Comment by WesCossick** [comment]
Tabs are useful for indented code blocks and for indenting list items more (tab) or less (shift-tab).

**Comment by fredericseiler** [comment]
``` js
var editor = new SimpleMDE();

editor.codemirror.options.extraKeys['Tab'] = false;
editor.codemirror.options.extraKeys['Shift-Tab'] = false;
```
